### ISSUE TITLE
Compare different models and methods for our Q&A-bot

### ISSUE THREAD

**Issue by egeatmaca** [egeatmaca/Legalpha#7]
We have a dataset mapping questions to answers in the data folder.
Research and conceptualize different models / methods to predict answers to given questions. 
Compare their fit to our use case qualitatively. 
Implement the suitable ones and compare the metrics.

Models / methods to implement: 
- Semantic Search
- Question Answering Models (Fine-tuned BERT)
- Classification Models (Neural Networks, XGBoost, LightGBM, Fine-tuned BERT)

Metrics to compare: 
- Accuracy
- Recall
- Precision
- F1-score
- Log Loss

**Comment by egeatmaca** [comment]
This issue depends on issue #8 for resolution.
